src/condition_variable.hpp

```diff
@@ -214,7 +214,7 @@ namespace zmq
                 struct timespec timeout;
 
 #if defined ZMQ_HAVE_OSX && __MAC_OS_X_VERSION_MIN_REQUIRED < 101200 // less than macOS 10.12
-                alt_clock_gettime(CLOCK_REALTIME, &timeout);
+                alt_clock_gettime(CALENDAR_CLOCK, &timeout);
```

Comment:
thanks - this is a wrapper around pthread conditions, so monotonic clock is the right thing